go.mod

```diff
@@ -1,9 +1,169 @@
 module github.com/mackerelio-labs/terraform-provider-mackerel
 
-go 1.15
+go 1.17
```

Comment:
Since testing with go 1.16 results in the following error, we suggest defining a minimum version to test (= `1.15`).

```
go mod tidy: go.mod file indicates go 1.17, but maximum supported version is 1.16
```